packages/grant-explorer/src/features/round/__tests__/PassportConnect.test.tsx

```diff
@@ -146,8 +147,8 @@ describe("<PassportConnect/>", () => {
         json: () => mockJsonPromise,
       } as unknown as Response;
 
-      (fetchPassport as jest.Mock).mockResolvedValueOnce(mockPassportPromise);
-      (submitPassport as jest.Mock).mockResolvedValueOnce(jest.fn());
+      (fetchPassport as Mock).mockResolvedValueOnce(mockPassportPromise);
+      (submitPassport as Mock).mockResolvedValueOnce(vi.fn());
```

Comment:
Not part of this PR, but this looks slightly suspicious — does `submitPassport` actually return a function?

Reply:
Not sure, but OOS for this PR.